src/components/visx/TimeSeriesChart.tsx

```diff
@@ -228,8 +232,13 @@ export const TimeSeriesChart = <Datum extends {}>({
     // TODO: scroll horizontally to pan
   };
 
+  useEffect(() => {
+    // Prevents scrolling of the page when user is hovered over chart (scrolling should adjust zoom of the chart instead)
+    chartRef.current?.addEventListener('wheel', (e) => e.preventDefault());
```

Comment:
should we remove this listener or is it fine because it'll be unmounted?

Reply:
oh interesting why can't this go in the onWheel?

Reply:
It'll fail with `Unable to preventDefault inside passive event listener invocation` because React uses passive event [handlers](https://stackoverflow.com/questions/63663025/react-onwheel-handler-cant-preventdefault-because-its-a-passive-event-listenev) with `wheel` events. good point on removing though!